include/manifold/manifold.h

```diff
@@ -46,11 +46,59 @@ class CsgLeafNode;
  * @brief Mesh input/output suitable for pushing directly into graphics
  * libraries.
  *
- * This may not be manifold since the verts are duplicated along property
- * boundaries that do not match. The additional merge vectors store this missing
- * information, allowing the manifold to be reconstructed. MeshGL is an alias
- * for the standard single-precision version. Use MeshGL64 to output the full
- * double precision that Manifold uses internally.
+ * The core (non-optional) parts of MeshGL are the triVerts indices buffer and
+ * the vertProperties interleaved vertex buffer, which follow the conventions of
+ * OpenGL (and other graphic libraries') buffers and are therefore generally
+ * easy to map directly to other applications' data structures.
+ *
+ * The triVerts indices should form a manifold mesh: each of the 3 halfedges of
```

Comment:
Should we specify what halfedges are? For beginners they may not be aware of this.